pgcommitfest/commitfest/views.py

```diff
@@ -86,14 +86,44 @@ def me(request):
     if patch_list.redirect:
         return patch_list.redirect
 
+    # Get stats related to user for current commitfest
+    curs = connection.cursor()
+    curs.execute(
+        """SELECT
+            ps.status, ps.statusstring, count(*)
+        FROM commitfest_patchoncommitfest poc
+        INNER JOIN commitfest_patch p ON p.id = poc.patch_id
+        INNER JOIN commitfest_patchstatus ps ON ps.status=poc.status
+        WHERE
+            ps.status NOT IN ( %(patch_stat_committed)s, %(patch_stat_rejected)s, %(patch_stat_withdrawn)s, %(patch_stat_nextcf)s, %(patch_stat_retwfdb)s )
```

Comment:
If you look at the other query you'll see that it uses the following filter to achieve the same:

```suggestion
            ps.statusstatus = ANY(%(openstatuses)s)
```

Let's use that here too, to stay consistent. It's also much shorter.

Reply:
oops! missed that 😅